print/templates/pattern_only.html

```diff
@@ -0,0 +1,25 @@
+    <main class="page pattern-onlyview" id="pattern-display">
+      <figure class="inksimulation pattern-only operator"
+              data-field-name="pattern-onlyview"
+              title="{{ _('Ctrl + Scroll to Zoom') }}"
+              style="cursor: auto;">
+                {{ svg_overview|replace("<li>", "")|replace("</li>", "")|safe }}
+                {% with %}
+                {% set realistic_id='realistic-operator-overview' %}
+                {% endwith %}
+                <div>
+                  <button class="svg-fit">{{ _('Fit') }}</button>
+                  <button class="svg-full">100%</button>
+                  <button class="svg-apply">{{ _('Apply to all') }}</button>
+                  <button class="svg-realistic">
+                    {% if loop_index is defined %}
+                    <input type="checkbox" id="realistic-color-block-{{ loop_index }}" data-field-name="block{{ loop_index }}" class="realistic" />
+                    <label for="realistic-color-block-{{ loop_index }}" class="realistic">{{ _('Realistic') }}</label>
+                    {% else %}
+                    <input type="checkbox" id="{{ realistic_id }}" data-field-name="overview" class="realistic" />
+                    <label for="{{ realistic_id }}" class="realistic">{{ _('Realistic') }}</label>
+                    {% endif %}
+                  </button>
```

Comment:
Remove the div with the buttons. We have a template for this (see above)

Reply:
The reason I did this was to remove the figcaption that is also included in the ui_svg_action_buttons because I don't think the scale should be printed out if you're just trying to view the pattern. 

Any other way around this?

Reply:
You could display it at first (could be still interesting while setting it up), but exclude it from the output with css.

In style.css in line 1179 (@media print) you can add

```

    div.pattern-onlyview figcaption {
        display: none;
    }
```